src/utils/hotspotUtils.ts

```diff
@@ -37,10 +37,11 @@ export const generateRewardScaleColor = (rewardScale: number): Colors => {
   return 'redMain'
 }
 
-export const isRelay = (listenAddrs: string[]) => {
+export const isRelay = (listenAddrs: string[] | undefined) => {
+  if (!listenAddrs) return false
   const IP = /ip4/g
 
-  return !!(
+  return (
     listenAddrs &&
```

Comment:
I think `listenAddrs &&` can safely be removed now that you added the early return.